packages/trees/frontend/sirius-components-trees/src/views/__tests__/GraphqlTreeEventSubscription.test.ts

```diff
@@ -40,7 +40,22 @@ const getDocumentSubscription = gql`
     id
     hasChildren
     expanded
-    label
+    label {
+      styledStringFragments {
+        text
+        styledStringFragmentStyle {
+          isStrikedout
```

Comment:
I won't make you change it now but I don't think that `isStrikedOut` is valid in English. I think that it should be `isStruckOut`. We will change it later